src/api/route-services/pipeline-response.js

```diff
@@ -72,7 +73,7 @@ const pipelineResponse = async (io, message) => {
       plotsTableService.updatePlotData(
         experimentId,
         plotUuid,
-        output.plotData,
+        output.plotDataKeys[plotUuid],
```

Comment:
Here this says `plotDataKeys` but in other places it says `plotDataKey`. What is the difference between them?

Reply:
I am calling `plotsTableService.updatePlotData()` with one of the values in `plotDataKeys` (`plotDataKeys[plotUuid]`), so inside `plotsTableService.updatePlotData()`  I decided to call this value `plotDataKey.`